lib/templates/pullRequestOpened.hbs

```diff
@@ -21,6 +21,11 @@ Can you please send in a [Contributor License Agreement](https://github.com/Anal
 @{{ userName }}, thanks for the pull request!
 {{/if}}
 
+{{#if askAboutContributors}}
+Don't forget to add your name to the `CONTRIBUTORS.md` file!  
```

Comment:
This seems generic, I think it should be more direct. Something like,

"It looks like you have not added yourself to `CONTRIBUTORS.md`. Please add yourself and confirm by commenting on this pull request."